app/src/test/java/com/futurice/freesound/feature/home/HomeFragmentViewModelTest.kt

```diff
@@ -81,29 +82,7 @@ class HomeFragmentViewModelTest {
     private fun createVm(): HomeFragmentViewModel =
             HomeFragmentViewModel(userDataModel)
 
-
-    private class UserBuilder {
-
-        internal lateinit var largeAvatar: String
-        internal lateinit var userName: String
-        internal lateinit var about: String
-
-        internal fun build() = mock<User>().apply {
-            val avatar = mock<Avatar>()
-            `when`(avatar.large()).thenReturn(largeAvatar)
-            `when`(avatar()).thenReturn(avatar)
-            `when`(username()).thenReturn(userName)
-            `when`(about()).thenReturn(about)
-        }
-    }
-
-    companion object {
-
-        private val USER = UserBuilder()
-                .apply {
-                    largeAvatar = "large.com"
-                    userName = "user name"
-                    about = "aboutTextView"
-                }.build()
+    fun ignore(ignoreReturnValue: () -> Unit) {
```

Comment:
Let's keep these in mind so that we can "library" a set of such methods (similarly with `KotlinTest.kt`). (Also reminds me about having something to convert your Unit to Kotlin's Unit. Perhaps we can get them to include yours in the compiler checks so it behaves the same way :-)